bundles/core-bundle/post-install.d/04-remember-previous-secc-version

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#
+# This script copies over the current firmware information to the updated system
+# so that (version dependend) post-processing of configuration files will be
+# possible after the restart of the system.
+#
+
+SLOT_CLASS="$1"
+SLOT_MOUNT_POINT="$2"
+BUNDLE_MOUNT_POINT="$3"
+
+SOURCE_DIR="/usr/share/secc"
+TARGET_DIR="/var/lib/secc/previous-version"
+
+if [ "$SLOT_CLASS" = "rootfs" ]; then
+	# create destination directory
+	mkdir -p "$SLOT_MOUNT_POINT/$TARGET_DIR"
+
+	# copy version information files if present
+	ls "$SOURCE_DIR"/* &> /dev/null && \
+		cp "$SOURCE_DIR"/* "$SLOT_MOUNT_POINT/$TARGET_DIR"
```

Comment:
I never liked the `ls` solution, but okay by me.
Can we preserve timestamps by using `cp -p`? (Not important in this case, but helpful in some others.)

Wouldn't this potentially suffer the same run-parts return code 2 message, if `ls` fails? (I know, should never happen.)